packages/fxa-auth-server/lib/customs.js

```diff
@@ -306,6 +306,18 @@ class CustomsClient {
       return false;
     }
 
+    // The config can specify that certain ips, emails, or uids should be excluded
+    // from rate limit checks.
+    const skip = this.rateLimit.skip(opts);
+    if (skip) {
+      this.statsd.increment(`${serviceName}.check.v2.skip`, [
+        opts.ip ? 'ip':'',
+        opts.email ? 'email':'',
+        opts.uid ? 'uid':'',
+      ]);
```

Comment:
This can cause a cardinality issue, depending on the configuration and environment where this is used.  E.g. if the email pattern is 'restmail'.  Do you think we'd want to know the specific ip/email/uid skipped?

Reply:
We are no passing in the IP, email or uid values themselves. Let's say all three opts were provided. The values passed in would be, ['email','ip','uid']. I don't think this would cause a cardinality problem.

Reply:
Oh I totally misread that.  👍